Dockerfile

```diff
@@ -61,6 +63,8 @@ RUN source activate cudf && \
     make install_python
 
 # cuDF build/install
-RUN source activate cudf && \
-    cd /cudf/python && \
-    python setup.py install
+#RUN source activate cudf && \
+#    cd /cudf/python && \
+#    python setup.py install
+RUN conda install -c nvidia -c rapidsai -c numba -c conda-forge -c defaults cudf=0.3.0
```

Comment:
We shouldn't install from conda in the container as this then ignores the branch specified by an arg earlier and prevents using the container for development.